doc/source/notebooks/basic.ipynb

```diff
@@ -86,7 +116,15 @@
             "source": [
                 "import greenplumpython as gp\n",
                 "\n",
-                "db = gp.database(host=\"localhost\", dbname=\"gpadmin\")"
+                "db = gp.database(uri=\"postgresql://localhost/gpadmin\")"
+            ]
+        },
+        {
+            "attachments": {},
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "Here, the `uri` follows the same specification in the [libpq document](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) as above."
```

Comment:
This is good enough. For me, it would be better to give a simple example with user/password, e.g.
postgresql://[user]:[password]@[host][/dbname]